File: packages/java/src/main/java/com/tesote/sdk/errors/SyncRateLimitExceededException.java
package com.tesote.sdk.errors;

/**
 * 429 — per-bank-connection sync was triggered too soon after the previous
 * one. Distinct from the per-API-key {@link RateLimitExceededException}.
 */
public final class SyncRateLimitExceededException extends ApiException {
    public SyncRateLimitExceededException(String message, String errorCode, int httpStatus,
                                          String requestId, String errorId, Integer retryAfter,
                                          String responseBody, RequestSummary requestSummary,
                                          int attempts, Throwable cause) {
        super(message, errorCode, httpStatus, requestId, errorId,
                retryAfter, responseBody, requestSummary, attempts, cause);
    }
}

File: packages/csharp/src/Errors/SyncRateLimitExceededException.cs
using System;

namespace Tesote.Sdk.Errors;

/// <summary>429 SYNC_RATE_LIMIT_EXCEEDED — bank-connection sync rate limit hit.</summary>
public sealed class SyncRateLimitExceededException : ApiException
{
    /// <summary>Construct with the full required-field set.</summary>
    public SyncRateLimitExceededException(
        string? message, string? errorCode, int httpStatus,
        string? requestId, string? errorId, int? retryAfter,
        string? responseBody, RequestSummary? requestSummary,
        int attempts, Exception? cause)
        : base(message, errorCode, httpStatus, requestId, errorId,
               retryAfter, responseBody, requestSummary, attempts, cause)
    {
    }
}
